.github/scripts/rancher-grant-cluster-access.sh

```
@@ -326,47 +355,48 @@ update_group_binding() {
     ] | .[]
   ' <<<"$json")
-  for id in "${ids[@]}"; do
-    [[ -n "$id" ]] || continue
-    principal="$(jq -r --arg id "$id" '[.data[]? | select(.id == $id)][0].userPrincipalId // empty' <<<"$json")"
-    log "Removing misbound user clusterRoleTemplateBinding id=${id} userPrincipalId=${principal}"
-    delete_binding_by_id "$id" || err "Failed to delete binding ${id}"
-  done
 }
 
 reconcile_stale_group_bindings() {
   local target="$1" json rows id name principal role
```

Shellcheck (0.11.0)

[warning] rows appears unused. Verify use (or export if used externally).

(SC2034)